### PR TITLE
Adding video thumbnail for the .mp4 extension

### PR DESCRIPTION
**Pull request by iFlameing**

### CODE REVIEW

**Review by tisto — APPROVED** [review]

File: src/components/manage/Blocks/Video/Edit.jsx [hunk 1]
```diff
@@ -253,6 +253,15 @@ class Edit extends Component {
                             : data.url
                         }
                         controls
+                        poster={
+                          data.preview_image
+                            ? isInternalURL(data.preview_image)
+                              ? `${flattenToAppURL(
+                                  data.preview_image,
+                                )}/@@images/image`
```
Comment (sneridagh) [inline src/components/manage/Blocks/Video/Edit.jsx:261]: This is wrong, right?

File: src/components/manage/Blocks/Video/View.jsx [hunk 2]
```diff
@@ -128,6 +128,15 @@ const View = ({ data }) => (
                         : data.url
                     }
                     controls
+                    poster={
+                      data.preview_image
+                        ? isInternalURL(data.preview_image)
+                          ? `${flattenToAppURL(
+                              data.preview_image,
+                            )}/@@images/image`
```
Comment (sneridagh) [inline src/components/manage/Blocks/Video/View.jsx:136]: same here

**Review by sneridagh — CHANGES_REQUESTED** [review]

File: src/components/manage/Blocks/Video/Edit.jsx (hunk 1 above)
Comment (iFlameing) [reply src/components/manage/Blocks/Video/Edit.jsx:261]: No, We are using it everywhere.
https://github.com/plone/volto/blob/master/src/components/manage/Blocks/Image/Edit.jsx#L268

File: src/components/manage/Blocks/Video/View.jsx (hunk 2 above)
Comment (iFlameing) [reply src/components/manage/Blocks/Video/View.jsx:136]: same
https://github.com/plone/volto/blob/master/src/components/manage/Blocks/Image/Edit.jsx#L268

File: src/components/manage/Blocks/Video/Edit.jsx (hunk 1 above)
Comment (iFlameing) [reply src/components/manage/Blocks/Video/Edit.jsx:261]: For large image

**Review by sneridagh — APPROVED** [review]

**Pull request merged**